### ISSUE TITLE
Add remote functionality

### ISSUE THREAD

**Issue by jasonroth** [michalmillar/ps-motd#3]
Hi,

I'd like to use your awesome function as part of a module I'm writing that provides  an operations menu for service desk folks. For this to work as envisioned, I need to add remote capabilities to your function. Do you mind if I do a PR?

https://github.com/jasonroth/ServerOpsMenu

**Comment by michalmillar** [comment]
Absolutely! I'm in the process of rewriting this as a module, so some things may change -but I am very excited to see what you do with this!

**Comment by michalmillar** [comment]
Wow, your rewrites look awesome!

**Comment by michalmillar** [comment]
Okay, I went ahead and merged your changes. I then went back and fiddled with the way it works. Give the script a try now and let me know if it is still working the way you are expecting it to.
